### DISCUSSION TITLE
[2514960] Refind Self: The Personality Test Game [仅本体]

### DISCUSSION THREAD

**Discussion by egnosyz** [pjy612/SteamManifestCache#32] [Share]
```
appid:2514960
keys:
2514961;cb0c853fe1fe32bb017a5a1bb05158edc251190653ad84e31745b7fefdbc7866
```
[2514961_2790101935117325786.zip](https://github.com/pjy612/SteamManifestCache/files/13682210/2514961_2790101935117325786.zip)  

code: 87EB599964097B78F20A0B20F3238FB9E5D15173
email: egnosyz@outlook.com

**Comment by egnosyz** [comment]
[1859280] 7 Days to End with You
```
appid:1859280
keys:
1859281;61c6b0b4486a044499b6aae9a4e8e212360700d7b34feec5f435cb4d691dc5a2
```

**Comment by pjy612** [comment]
感谢分享请查收邮件
